### ISSUE TITLE
Generate button lags

### ISSUE THREAD

**Issue by ahlamjaved** [TailwindEngineering/untangle-community#21]
All the generate buttons are extremely slow, needs enhancing.

**Comment by ahlamjaved** [comment]
No punctuation when using generate button![image](https://github.com/user-attachments/assets/1e19ca18-4405-483b-bef5-b37f1873c8ef)

**Comment by moorea** [comment]
You're right, it's slow. At the moment, we are not paying to keep the servers ready at all times. Since there are so few requests to the servers right now, you're having to wait a few seconds for it to spin up to service your request (also known as a cold start)

When we go to production, we will pay to ensure that at least 1 instance of the server is always ready.

That will make it much snappier and get us 90% of the way there.

The other thing we plan to do, is stream back the response instead of waiting for all the text to be generated before displaying it. When you tap the generate button, instead of showing the loading animation, we'll immediately stream back partial text as it becomes available. This will make it feel as though the app is typing for you and make the generate feel immediate.
